packages/build/src/plugins_core/secrets_scanning/index.ts

```diff
@@ -51,13 +53,15 @@ const coreStep: CoreStepFunction = async function ({
     log(logs, `SECRETS_SCAN_OMIT_PATHS override option set to: ${envVars['SECRETS_SCAN_OMIT_PATHS']}\n`)
   }
 
-  const keysToSearchFor = getSecretKeysToScanFor(envVars, passedSecretKeys)
+  const explicitSecretKeysToScanFor = getSecretKeysToScanFor(envVars, passedSecretKeys)
+  const otherKeysToScanFor = enhancedSecretScan ? getNonSecretKeysToScanFor(envVars, passedSecretKeys) : []
```

Comment:
NIT: could this have more meaningful name
(bad name, probably shouldn't use, but just as example) - `envVarsLookingLikeSecretsToScanFor` for variable and something similar for `get[...]` function

not a blocker - just to make browsing through code easier for people not fully knowing details of secret scanning to not have to look at details of function to understand on high level what's this is actually for

Reply:
yessss that's a good point, I'll change!